runtime/builtin_types_test.go

```diff
@@ -23,6 +23,33 @@ import (
 	"testing"
 )
 
+func TestBuiltinDelAttr(t *testing.T) {
+	fun := wrapFuncForTest(func(f *Frame, args ...*Object) (*Object, *BaseException) {
+		fooObject := newTestClass("Foo", []*Type{ObjectType}, toDictUnsafe(args[0])).ToObject()
+		result, raised := builtinDelAttr(f, append(Args{fooObject}, args[1:]...), nil)
```

Comment:
I would fetch "delattr" from the builtins dict and call it, rather than calling the underlying function directly. That will exercise the dispatch logic and ensure that the "delattr" key is in builtins.